### DISCUSSION TITLE
'poetry version minor' duplicating plugins in toml

### DISCUSSION THREAD

**Discussion by Nathaniel-Rink** [python-poetry/poetry#3554] [Q&A] [answered]
Good morning. I asked this question on [stack overflow](https://stackoverflow.com/questions/65621150/poetry-version-command-is-duplicating-code-in-pyproject-toml?noredirect=1#comment116029299_65621150) and they directed me to this forum. I'm new to poetry and using poetry 1.0.10

I'm using **poetry version minor** to increment the version. I thought this would only affect the version in my pyproject.toml, but it's also duplicating my `[tool.poetry.plugins]` and moving my `[build-system]` within the .toml. The first breaks it, the second is unexpected behavior but not a big concern.

Am I misunderstanding the use of `poetry version`?

Thanks for any help!

**Comment by finswimmer** [comment]
Hey @Nathaniel-Rink,

could you please provide a minimal example of a `pyproject.toml`, so one can reproduce the issue?

fin swimmer

**Reply by Nathaniel-Rink** [reply]
Thanks @finswimmer - 

The fix was to move the `[build-system]` into the location in the pyproject it was being auto-moved to and then the duplication stopped. I guess it's just doing more under the hood than I expected. But my problem is resolved. Thanks!

**Reply by finswimmer** [reply]
Could you please provide a `pyproject.toml` anyway. The behavior you've described doesn't sound like it should behave.

**Comment by Nathaniel-Rink** [comment, marked as answer]
The fix was to move the [build-system] into the location in the pyproject it was being auto-moved to and then the duplication stopped. If someone wants to elaborate why this is the case I'll mark that as the answer, otherwise I'll close this
